docs/platform-external/installing.md

```diff
@@ -0,0 +1,736 @@
+  OpenShift configuration files and specific customizations for platform type
+  `External`.
+- Section 2 - Infrastructure provisioning: the section describes how to consume
+  the configuration files, with an overview of the User-Provided Infrastructure (UPI)
+  installation method used by installations in non-integrated providers.
+- Section 3 - Setup Cloud Controller Manager (CCM): the section describes how to
+  create the required cloud provider resources when opted into it in the bootstrap
+  stage.
+
+## Table of Contents
+
+- [Prerequisites](#prerequisites)
+- [Section 1. Setup OpenShift configuration](#section-1-setup-openshift-configuration)
+    - [Create the install-config.yaml](#create-the-install-configyaml)
+    - [Create the manifests](#create-manifests)
+        - [Customize manifest for CCM (optional)](#customize-manifest-for-ccm-optional)
+            - [Patch the Infrastructure Object](#patch-the-infrastructure-object-optional)
+            - [Create MachineConfig for Kubelet Provider ID](#create-machineconfig-for-kubelet-provider-id-optional)
+    - [Create ignition files](#create-ignition-files)
+- [Section 2. Create Infrastructure resources](#section-2-create-infrastructure-resources)
+    - [Identity](#identity)
+    - [Network](#network)
+    - [DNS](#dns)
+    - [Load Balancers](#load-balancers)
+    - [Create compute nodes](#create-compute-nodes)
+        - [Upload the RHCOS image](#upload-the-rhcos-image)
+        - [Bootstrap](#bootstrap-node)
+        - [Control Plane](#control-plane)
+        - [Compute/workers](#computeworkers)
+- [Section 3. Deploy Cloud Controller Manager (CCM)](#section-3-deploy-cloud-controller-manager-ccm)
+- [Review the installation](#review)
+- [Next](#next-steps)
+
+The following sections describe the steps to install an OpenShift cluster using
+platform external type using fully customized automation.
+
+
+## Prerequisites
+
+### Clients
+
+The OpenShift client (`oc`) and Installer (`openshift-install`) must be
+downloaded, read the steps described in
+["Obtaining the installation program"](https://docs.openshift.com/container-platform/4.13/installing/installing_platform_agnostic/installing-platform-agnostic.html).
+
+The credentials to pull OpenShift images, also known as "Pull Secret", needs
+to be obtained from the [Red Hat Hybrid Cloud Console](https://console.redhat.com/openshift).
+
+## Section 1. Setup OpenShift configuration
+
+The OpenShift cluster configuration to set up the platform external is created in this section.
+
+The steps below describe how to create the `install-config.yaml` with the required fields, and
+the steps used to customize the providers' manifests before generating the ignition
+configuration files.
+
+### Create the install-config.yaml
+
+Follow the steps to [Manually create the installation configuration file](install-config),
+customizing the `platform` object, setting the type to `external`, and the
+`platformName` to the cloud provider's name:
```

Comment:
it would be nice to have. I added the following paragraph based in the API definition:
> The value of `platformName` holds the arbitrary string representing the infrastructure
provider name, expected to be set at the installation time. This field is solely for
informational and reporting purposes and is not expected to be used for decision-making.

Let me know if would be modified.